app/code/Transoft/Blog/Block/Adminhtml/Post/Edit/BackButton.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare(strict_types=1);
```

Comment:
Could you please add blank line after *stric_types* declaration. Please fix in **ALL** files

Reply:
added for every file